src/main/webapp/js/instructorFeedbackResultsAjaxByGQR.js

```diff
@@ -28,35 +27,44 @@ $(document).ready(function() {
                 displayAjaxRetryMessageForPanelHeading(displayIcon);
             },
             success: function(data) {
+                var sectionBody = panelBody[0];
```

Comment:
A minor point, but let's improve this a step further by directly `var $sectionBody = $(panelBody[0])`